src/modules/dashboard/transformers.ts

```diff
@@ -79,6 +83,30 @@ export const getActionsListData = (
               parseInt(`${timestamp}000`, 10),
             );
           }
+          if (subgraphActionType === 'events') {
+            const {
+              args,
+              associatedColony: {
+                token: { address: tokenAddress, symbol, decimals },
+              },
+              name,
+            } = unformattedAction;
+            const actionEvent = Object.entries(ACTIONS_EVENTS).find((el) =>
+              el[1]?.includes(name.split('(')[0]),
+            );
```

Comment:
Maybe have a util for this at some point, since I think we're using this in two places now.

But you can leave it like this for now, maybe I'll fix it when I'll wire in the rest of the events